localization/en-us.lua

```diff
@@ -2449,7 +2449,7 @@ return {
                 name = "Mass Production",
                 text = {
                     "All cards and packs",
-                    "in shop cost {C:attention}$1{}",
+                    "in the shop cost {C:attention}$1{}",
```

Comment:
I would prefer this not be changed since it isn't consistent with the wording used for Clearance Sale and Liquidation
(Blame Thunk)